app/src/main/res/layout/app_widget_classic.xml

```diff
@@ -3,17 +3,19 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/content"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="match_parent"
     android:background="@drawable/card"
     android:orientation="horizontal"
     tools:ignore="ContentDescription">
 
     <ImageView
         android:id="@+id/image"
-        android:layout_width="@dimen/app_widget_classic_image_size"
-        android:layout_height="@dimen/app_widget_classic_image_size"
-        android:background="@drawable/default_album_art"
-        android:scaleType="centerCrop" />
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:scaleType="centerInside"
```

Comment:
I guess I prefer `centerCrop` so it takes up the height? It's sort of wonky with the border radii  and the varying height though, I guess.